src/Artifact/UploadDisplay.jsx

```diff
@@ -597,12 +598,12 @@ function parseMainStatvalue(recognition, defVal = { mainStatValue: NaN }) {
   let texts = recognition?.data?.lines?.map(line => line.text)
   if (!texts) return defVal
   for (const text of texts) {
-    let regex = /(\d+\.\d)%/
+    let regex = /(\d+[,|\\.]+\d)%/
     let match = regex.exec(text)
-    if (match) return { mainStatValue: parseFloat(match[1]), unit: "%" }
+    if (match) return { mainStatValue: parseFloat(match[1].replace(/,/g,".").replace(/\.{2,}/g, ".")), unit: "%" }
     regex = /(\d+,\d{3}|\d{2,3})/
```

Comment:
this only works with `X,XXX` and `XXX`, but would not work for `X.XXX`

Reply:
whoops missed, fixed on last commit